ui/tests/e2e/env.js

```diff
@@ -1,12 +1,12 @@
 const MESHERY_SERVER_URL = process.env.MESHERY_SERVER_URL || 'http://localhost:9081';
-const REMOTE_PROVIDER_URL = process.env.REMOTE_PROVIDER_URL || 'https://staging-meshery.layer5.io';
+const REMOTE_PROVIDER_URL = process.env.REMOTE_PROVIDER_URL || 'https://meshery.layer5.io';
 
 const USER_EMAIL = process.env.REMOTE_PROVIDER_USER_EMAIL;
 const USER_PASSWORD = process.env.REMOTE_PROVIDER_USER_PASSWORD;
 
 const REMOTE_PROVIDER_USER = {
-  email: USER_EMAIL || 'test-admin@layer5.io',
-  password: USER_PASSWORD || 'test-admin',
+  email: USER_EMAIL,
```

Comment:
Nice! I was curious about these defaults!

Reply:
@ianrwhitney is just an old account I guess which is already not valid anymore